tests/wpt/mozilla/meta-layout-2020/mozilla/sequence-hole.html.ini

```diff
@@ -2,6 +2,3 @@
   [Sequences with holes and a getter on the prototype are correctly handled]
```

Comment:
We might need to turn on `prefs: [dom.testbinding.enabled:true]` in this file 👀?